config/constants.go

```diff
@@ -5,8 +5,8 @@ import "path/filepath"
 const (
 	// StaticVersionNumber is the version of Owncast that is used when it's not overwritten via build-time settings.
 	StaticVersionNumber = "0.1.1" // Shown when you build from develop
-	// FfmpegSuggestedVersion is the version of ffmpeg we suggest.
-	FfmpegSuggestedVersion = "v4.1.5" // Requires the v
+	// FfmpegWithGeneralPixFmtVersion is the version of ffmpeg user for 'vaapi' fix_fmt value, instead of 'vaapi_vld'.
+	FfmpegWithGeneralPixFmtVersion = "v5.0" // Requires the v
```

Comment:
I know we had that ffmpeg version in constants before. The thought was it would be changed over time, but that never really happened. The ffmpeg version pixel format version will never need to be changed, so maybe it should just live local to the single place it's being used.

Reply:
I check it and found usage in several places: codecs and tests, so i refactored usage